### DISCUSSION TITLE
Where are Settings Stored

### DISCUSSION THREAD

**Discussion by kevinrjones** [russhwolf/multiplatform-settings#152] [General]
Hi Russell,

I'm sure I should be able to figure this out. I'm using the default settings (val settings = Settings()), running a ComposeDesktop app and the settings are getting saved and survive an app restart. I'm running on a Mac and I was wondering where the values are stored. I've looked in what I think are the places I'd expect to find them and don't see them,

thanks

**Comment by kevinrjones** [comment]
Ah, found them, on the Mac they're in ~/Library/Preferences/com.apple.java.util.prefs.plist

**Comment by jamesmbowler** [comment]
I see that file, but I don't see my data .. even though my app is telling me it's there. Any tips?

**Reply by jamesmbowler** [reply]
got it .. Library/Developer/CoreSimulator/Devices/6DEC18E3-5AF6-460F-8787-0C482150221C/data/Containers/Data/Application/EB6469BD-D3EF-4535-80C2-198D7292AC9F/Library/Preferences
